src/Colors.tsx

```diff
@@ -0,0 +1,12 @@
+export const Colors = {
```

Comment:
@pedromvpg Can you add a complete list of colors used in the design? Probably best in a separate PR, but 🤷‍♂️

Reply:
as discussed @pedromvpg may be cleaing up / paring down colors used in designs.  With the latest commit I have pulled in all the colors (I think) used in the designs.